model/task/db.go

```diff
@@ -160,6 +160,32 @@ var (
 		},
 	}
 
+	sortDisplayName = bson.M{
```

Comment:
Can we add comments to explain this whole section?

Reply:
`sortByDisplayName`?